app/controllers/plants_controller.rb

```diff
@@ -1,2 +1,10 @@
 class PlantsController < ApplicationController
+  def show
+    @plants = Trail.find(params[:id]).region.plants
+    render 'show.json.jbuilder'
```

Comment:
By default, you don't need to do render 'index.json.jbuilder' for index, show, new and edit actions.